src/Nethermind/Nethermind.Api/Extensions/CompositePluginLoader.cs

```diff
@@ -50,5 +50,10 @@ public void Load(ILogManager logManager)
                 }
             }
         }
+
+        public void OrderPlugins(IPluginConfig pluginConfig)
+        {
+            throw new NotImplementedException();
```

Comment:
hmm?

Reply:
Is this class used anywhere? If not lets delete it